modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3799,6 +3800,7 @@ public CodegenResponse fromResponse(String responseCode, ApiResponse response) {
                 default:
                     throw new RuntimeException("Invalid response code " + responseCode);
             }
+            r.isSuccessCode = r.code.startsWith("2");
```

Comment:
Heads-up: later we may switch it to something like r.is2XXCode as 2 or 3 other generators need is4XXCode and is5XXCode as well (and probably is3XXCode)

Reply:
Good catch! That's actually already the case - see the `switch` statement right above this line. I'll fix it later today.